compile/annotation.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
@mihai-chiorean You should be able to use `scripts/updateLicenses.sh` to handle this.